packages/commonwealth/server.ts

```diff
@@ -125,9 +125,13 @@ async function main() {
   const setupMiddleware = () => {
     // redirect from commonwealthapp.herokuapp.com to commonwealth.im
     app.all(/.*/, (req, res, next) => {
+      if (req.ip.includes(process.env.DISABLE_REDIRECT_URL)) {
```

Comment:
We need safe defaults for these, in `config.ts`.

Reply:
Safe default should be undefined (which is what process.env.DISABLE_REDIRECT_URL would resolve to).

Unless you are thinking of some other default we should have for this?

Reply:
Undefined is ok for this one (unless `.includes(undefined)` acts weird?), but what about for the other two? They'll only be set on heroku.